_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,284 @@
+                <span class="input-label">Email at matttracker@gmail.com</span>
+            </label>
+            <div class="helper-text" id="updates_email_02_helper">We will never spam you. Pinky swear.</div>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Checkboxes</h4>
+<div class="example-set">
+    <fieldset class="fieldset form-group">
+        <legend class="group-legend form-label">Default checkboxes</legend>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="outsmart_prof_01">
+                <input type="checkbox" class="input input-checkbox" name="edx_goals_01" id="outsmart_prof_01" value="Outsmart those professors" checked="checked">
+                <span class="input-label">Outsmart those professors</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="try_and_see_01">
+                <input type="checkbox" class="input input-checkbox" name="edx_goals_01" id="try_and_see_01" value="Try it out and see what I can learn">
+                <span class="input-label">Try it out and see what I can learn</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="new_job_01">
+                <input type="checkbox" class="input input-checkbox" name="edx_goals_01" id="new_job_01" value="I need a new jobby job!" checked="checked">
+                <span class="input-label">I need a new jobby job!</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="fieldset form-group">
+        <legend class="group-legend form-label">Enhanced checkboxes</legend>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="outsmart_prof_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="outsmart_prof_02" value="Outsmart those professors" checked="checked">
+                <span class="input-label">Outsmart those professors</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="try_and_see_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="try_and_see_02" value="Try it out and see what I can learn">
+                <span class="input-label">Try it out and see what I can learn</span>
+            </label>
+        </div>
+        <div class="field-wrapper">
+            <label class="form-label inline-label" for="new_job_02">
+                <input type="checkbox" class="input input-checkbox replace-checkbox" name="edx_goals_02" id="new_job_02" value="I need a new jobby job!" checked="checked">
+                <span class="input-label">I need a new jobby job!</span>
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6">Textareas</h4>
```

Comment:
Add ``.example-set-hd`` class in here for consistency and extra PL Doc site formatting.